pkg/apis/eventing/v1beta1/broker_conversion_test.go

```diff
@@ -209,11 +209,11 @@ func TestBrokerConversionRoundTripV1(t *testing.T) {
 			t.Run(test.name, func(t *testing.T) {
 				ver := version
 				if err := ver.ConvertFrom(context.Background(), test.in); err != nil {
-					t.Errorf("ConvertDown() = %v", err)
+					t.Errorf("ConvertFrom() = %v", err)
```

Comment:
Irrelevant to subscription conversion.
Just saw this and wanted to fix it

Reply:
how about different PR, for better scope ?

Reply:
yeah, removed from here